wp-content/themes/core/assets/pcss/media/_utilities.pcss

```diff
@@ -0,0 +1,34 @@
+/* -------------------------------------------------------------------------
+ *
+ * Aspect Ratio Helpers
+ *
+ * Useful for fixing the height of an element based on it's width.
+ * Since the default Image block contains aspect ratio controls,
+ * these helpers should only be used if using an aspect ratio that doesn't
+ * exist within WP.
+ *
+ * Example:
+ * <figure class="wp-block-image s-aspect-ratio-1-1 s-aspect-ratio-cover">
+ *     <img class="wp-post-image" src="..."></img>
+ * </figure>
+ *
+ * ------------------------------------------------------------------------- */
+
+.s-aspect-ratio-cover,
+.s-aspect-ratio-cover > img {
+	height: 100%;
+	width: 100%;
+	object-fit: cover;
+}
+
+.s-aspect-ratio-1-1 {
+	aspect-ratio: 1 / 1;
+}
+
+.s-aspect-ratio-4-3 {
+	aspect-ratio: 4 / 3;
+}
+
+.s-aspect-ratio-16-9 {
+	aspect-ratio: 16 / 9;
+}
```

Comment:
Since these are no longer spacing selectors, probably should remove the `s-` prefix, yes?

Reply:
Removed prefix and changed aspect ratios to ratios that do not exist currently in the aspect ratio picker on the image block.